### PR TITLE
SOC-2120 | Custom 404 error for reported posts

### PR DESCRIPTION
**Pull request by bkoval**
## Links

* http://wikia-inc.atlassian.net/browse/JIRA-2120

## Description

We want to alter the default 404 message on reported-posts page.

## Reviewers

@kvas-damian @rafalkalinski

### CODE REVIEW

File: front/common/public/locales/en/discussion.json [hunk 1]
```diff
@@ -70,7 +70,8 @@
     "locked-post-text": "This post is locked.",
     "lock-post": "Lock Post",
     "unlock-post": "Unlock Post",
-    "locked-and-reported-to-moderators-text": "This post is locked, and has been reported."
+    "locked-and-reported-to-moderators-text": "This post is locked, and has been reported.",
+    "all-discussions-link-text": "Show me all discussions"
```
Comment (rafalkalinski) [inline front/common/public/locales/en/discussion.json]: what about all-discussions-link-desktop and all-discussions-link-mobile?

File: front/main/app/controllers/discussion/reported-posts-error.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend(
+	{
+		application: Ember.inject.controller(),
+
+		smartBannerVisible: Ember.computed.oneWay('application.smartBannerVisible'),
+		siteHeadPinned: Ember.computed.oneWay('application.siteHeadPinned'),
```
Comment (kvas-damian) [inline front/main/app/controllers/discussion/reported-posts-error.js]: Please use base error class to avoid repeating yourself.

**Comment by kvas-damian** [comment]
Please fix linter

**Comment by kvas-damian** [comment]
LGTM

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**